tasks/main.yml

```diff
@@ -10,6 +10,7 @@
 - include: default_site.yml
 - include: sites.yml
 - include: loadbalancers.yml
+- include: basic-auth.yml
```

Comment:
Would it make sense to have a check here so that we don't always include the basic-auth task when we are not using it? The task installs a package even when there are no basic auth files set.